corehq/apps/app_manager/static/app_manager/js/form_designer.js

```diff
@@ -0,0 +1,171 @@
+/* globals COMMCAREHQ, define, require, analytics, form_tour_start */
+hqDefine("app_manager/js/form_designer.js", function() {
+    $(function() {
+        var v2 = COMMCAREHQ.toggleEnabled('APP_MANAGER_V2'),
+            initial_page_data = hqImport("hqwebapp/js/initial_page_data.js").get;
+        var VELLUM_OPTIONS = _.extend(initial_page_data("vellum_options"), {
```

Comment:
I forget if it's safe to modify objects returned by `initial_page_data`. Like, does that modify global state? If it's not, then this should have `{}, ` before `initial_page_data`.